libs/datasets/combined_datasets.py

```diff
@@ -140,13 +140,16 @@ def datasource_regions(
 
 # NY Times has cases and deaths for all boroughs aggregated into 36061 / New York County.
 # Remove all the NYC data so that USAFacts (which reports each borough separately) is used.
-# Also, on 2/20, NYTimes stopped updating IA counties as well, removing from NYTimes dataset
-# to rely on USA Facts.
-NYTimesDatasetWithoutNYCOrIACounties = datasource_regions(
-    NYTimesDataset,
-    exclude=[RegionMask(level=AggregationLevel.COUNTY, states=["IA"]), *ALL_NYC_REGIONS],
+NYTimesDatasetWithoutNYC = datasource_regions(NYTimesDataset, exclude=[*ALL_NYC_REGIONS],)
```

Comment:
So Igor asserted that this problem is solved but I haven't looked at the data to confirm.  Kicked off a snapshot.  If it looks good, then I'll approve / merge.